_includes/version.html

```diff
@@ -1,8 +1,8 @@
 <div class="version-marker">
   <p>
     Latest versions:
-      <a href="{{ site.baseurl }}manuals/1.23/index.html#Releasenotesfor1.23">1.23.1</a>
+      <a href="{{ site.baseurl }}manuals/1.24/index.html#Releasenotesfor1.24">1.24.0</a>
       /
-      <a href="{{ site.baseurl }}manuals/1.22/index.html#Releasenotesfor1.22">1.22.2</a>
+      <a href="{{ site.baseurl }}manuals/1.23/index.html#Releasenotesfor1.23">1.23.1</a>
   </p>
```

Comment:
I think you can use `/latest`/ link: https://theforeman.org/manuals/latest/index.html

It currently redirect to 1.23 tho, we should fix this as we use this in the main menu.